tips/TIP-0028/event-api.yml

```diff
@@ -0,0 +1,1103 @@
+      transactionId:
+        description: Hex encoded identifier of the transaction.
+        schema:
+          type: string
+          example: "0xd026f8b1c856d4e844cc734bbe095429fb880ec4d93f3ccffe3b292a7de17be7"
+    subscribe:
+      tags:
+        - name: transactions
+      operationId: transactions
+      description: Publishes the confirmed block which carried the transaction with the specified transaction id.
+      message:
+        $ref: '#/components/messages/Block'
+  outputs/{outputId}:
+    parameters:
+      outputId:
+        description: Hex encoded identifier of the output.
+        schema:
+          type: string
+          example: "0xd026f8b1c856d4e844cc734bbe095429fb880ec4d93f3ccffe3b292a7de17be70000"
+    subscribe:
+      tags:
+        - name: outputs
+      operationId: output
+      description: Publishes the given wanted output on subscription and when its state changes (created, spent).
+      message:
+        $ref: '#/components/messages/OutputPayload'
+  outputs/alias/{aliasId}:
+    parameters:
+      aliasId:
+        description: The unique identifier of the alias chain. Hex encoded with 0x prefix.
+        schema:
+          type: string
+          example: "0x1505ec099896ab05d9e08fbc7101ae4dff0093b3943b28f789ed2ca728bcc8d6"
+    subscribe:
+      tags:
+        - name: outputs
+      operationId: aliasOutputByAliasId
+      description: Publishes the newly created alias output of an alias chain.
+      message:
+        $ref: '#/components/messages/OutputPayload'
+  outputs/nft/{nftId}:
+    parameters:
+      nftId:
+        description: The unique identifier of the nft chain. Hex encoded with 0x prefix.
+        schema:
+          type: string
+          example: "0x19c82b32761fd8729a1a6c77f7c17597e4b9b01759794e52381f6a0050b0c11f"
+    subscribe:
+      tags:
+        - name: outputs
+      operationId: nftOutputByNftId
+      description: Publishes the newly created nft output of an nft chain.
+      message:
+        $ref: '#/components/messages/OutputPayload'
+  outputs/foundry/{foundryId}:
+    parameters:
+      foundryId:
+        description: The unique identifier of the foundry chain. Hex encoded with 0x prefix.
+        schema:
+          type: string
+          example: "0x081505ec099896ab05d9e08fbc7101ae4dff0093b3943b28f789ed2ca728bcc8d60100000000"
+    subscribe:
+      tags:
+        - name: outputs
+      operationId: foundryOutputByFoundryId
+      description: Publishes the newly created foundry output of a foundry chain.
+      message:
+        $ref: '#/components/messages/OutputPayload'
+  outputs/unlock/{condition}/{address}:
+    parameters:
+      condition:
+        description: Type of unlock condition of the output to look for.
+        schema:
+          type: string
+          enum:
+            - "address"
+            - "storage-return"
+            - "expiration"
+            - "state-controller"
+            - "governor"
+            - "immutable-alias"
+            - "+"

```

Comment:
What is `+` ? Is it meant like "any" ? Maybe a more meaningful name ?

Reply:
it's a single level MQTT wildcard, read more about it here https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices/